app/src/main/java/com/example/bookwormadventuresdeluxe2/Book.java

```diff
@@ -131,44 +138,82 @@ public void deleteRequester(String requester)
         this.requesters.remove(requester);
     }
 
-    public String getBorrower()
-    {
-        return borrower;
-    }
-
-    public void setBorrower(String borrower)
-    {
-        this.borrower = borrower;
-    }
-
     /**
      * Sets the color of an image view based on the given status
      *
-     * @param status       The status of the item displaying the imageView
      * @param statusCircle The reference to the imageView to re-color
+     * @param user         The user to find the status for
      */
-    public void setStatusCircleColor(Status status, ImageView statusCircle)
+    public void setStatusCircleColor(ImageView statusCircle, String user)
     {
-        switch (status)
+        switch (getAugmentStatus(user))
         {
             case Available:
                 statusCircle.getDrawable().setColorFilter(ResourcesCompat.getColor(GlobalApplication.getAppContext().getResources(), R.color.available, null), PorterDuff.Mode.SRC_ATOP);
                 break;
-            case Borrowed:
-                statusCircle.getDrawable().setColorFilter(ResourcesCompat.getColor(GlobalApplication.getAppContext().getResources(), R.color.borrowed, null), PorterDuff.Mode.SRC_ATOP);
-                break;
             case Requested:
                 statusCircle.getDrawable().setColorFilter(ResourcesCompat.getColor(GlobalApplication.getAppContext().getResources(), R.color.requested, null), PorterDuff.Mode.SRC_ATOP);
                 break;
             case Accepted:
                 statusCircle.getDrawable().setColorFilter(ResourcesCompat.getColor(GlobalApplication.getAppContext().getResources(), R.color.accepted, null), PorterDuff.Mode.SRC_ATOP);
                 break;
+            case Borrowed:
+                statusCircle.getDrawable().setColorFilter(ResourcesCompat.getColor(GlobalApplication.getAppContext().getResources(), R.color.borrowed, null), PorterDuff.Mode.SRC_ATOP);

```

Comment:
Optionally, I think this line can be cut into multiple lines. Something like: 
![image](https://user-images.githubusercontent.com/43558584/97954489-e589a080-1d60-11eb-9b6c-133473c15e13.png)